### ISSUE TITLE
Feature Request: Main Search bar

### ISSUE THREAD

**Issue by cearum** [open-eats/OpenEats#67]
It would be nice to have a main search bar. Perhaps to the right of the Grocery List text or to the left of login/my account. That way recipes could be searched from any page and not just the browse page.

**Comment by RyanNoelk** [comment]
Good idea. When I get some time in the nest few weeks I will look at adding this.
